src/spec/unpeg-spec.ts

```diff
@@ -195,4 +211,24 @@ Error: Access Denied user Ke$ha *6* Name does not have the correct privileges
 			done();
 		})
 	});
+
+	it('should properly unpeg an abstract concept', async (done : DoneFn) => {
```

Comment:
Add a test to cover the case where the user doesn't add the 'for' keyword?

Reply:
done